game/entity/entity_enum.py

```diff
@@ -0,0 +1,5 @@
+from enum import Enum
+
+
+class Entities(Enum):
```

Comment:
It is (going to be) used for entity types. Instead of entity.type being string, it will be this enum. Version where this is implemented will be pushed soon, but onto the another branch